Topics/Product_Management/Product_Management_vs_Product_Ownership.md

```diff
@@ -0,0 +1,81 @@
+
+A product manager sits at the intersection of business, technology, and user experience. Product Managers need to balance all of those three needs and make hard decisions and trade-offs. Product Managers (PMs) are usually referred to as the "CEO of the product" because they set the goals, define success, help motivate teams, and are responsible for the outcome.
+
+## What are a Product Manager's responsibilities?
+
+In larger organizations, product managers are embedded within teams of specialists. Researchers, analysts, and marketers help gather input, while developers and designers manage the day-to-day execution, draw up designs, test prototypes, and find bugs. These product managers have more help, but they also spend more time aligning the stakeholders behind a specific vision.
+
+Product Managers at smaller organizations spend less time getting everyone to agree, but more time doing the hands-on work that comes with defining a vision and seeing it through.
+
+In general, a good product manager will spend their time on a handful of tasks such as:
+* Understanding and representing user needs.
+* Monitoring the market and developing competitive analyses.
+* Defining a vision or roadmap for a product
+* Aligning stakeholders around the vision for the product.
+* Prioritizing product features and capabilities.
+* Creating a cross-functional team to empower independent decision-making.
+
+## What is a Product Owner?
+
+A product owner is a Scrum team role that's accountable for the product's outcome and responsible for maximizing its value.
+
+As a member of the Scrum Team, the Product Owner provides clarity to the team about a product's vision and goal. All work is derived and prioritized based on the **Product Goal** in order to deliver value to all stakeholders including those within their organization and all users both inside and out. Product Owners (POs) identify, measure and maximize value throughout the entire product's lifecycle.
+
+## What are a Product Owner's responsibilities?
+
+The product owner is accountable for effective Product Backlog management, which includes:
+
+* Listening to customers and turning their problems and complaints into user stories that create actionable items. They prioritize these stories and then arrange them in the product backlog.
+* Building and setting production process priorities to make sure the development team knows specifically what thing to focus on in order of importance
+* The product owner is present at all team meetings with other internal stakeholders such as the scrum master to make sure that the whole development process is following the product roadmap created by the product manager
+* They are the communication link between the customer and the development teams. The product owner is the embodiment of the customer's voice
+* Giving constructibe feedback on the ratified roadmap. This is a communication link between the product owner and the product manager.
```

Comment:
This appears to be a typo here (constructibe), correcting such minor errors will improve the overall professionalism of the document.